src/style.css

```diff
@@ -0,0 +1,133 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500&family=Nunito:wght@400;700&family=Roboto:wght@400;500;700;900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset margins and paddings using `*` selector. Use individual tags if necessary.